src/components/input/input.css

```diff
@@ -0,0 +1,63 @@
+@import url('/src/components/utils.css');
+
+.container-signup{
+    border : 2px solid var(--border-color);
+    border-radius: 0.4rem;
+    display: flex;
+    justify-content: center;
+}
+.form{
+    display: flex;
+    flex-direction: column;
+    gap : 1rem ;
+    padding : 1.5rem ;
+}
+.input-field {
+    border: 1px solid var(--border-color);
+    border-radius: 5px ;
+    padding : 5px ; 
+    width: 100%;
+    font-size: 2.0rem;
+}
+.btn-primary{
+    width : 100% ;
+}
+.form-signup{
+    padding : 1.5rem ;
+    border :2px solid var(--ca-primary-color-lighter);
+    background-color: var(--white-color);
+    margin : 1rem ;
+    width : 350px ;
```

Comment:
can used rem instead of px

Reply:
i wanted to make a standard signup form, so did that

Reply:
but it is still reponsive